GenerateToolingFeed/V4Format/V4FormatFeedEntryUpdater.cs

```diff
@@ -45,27 +52,15 @@ public JObject GetUpdatedFeedEntry(JObject feed, CoreToolsInfo coreToolsInfo)
         {
             V4FormatFeedEntry feedEntry = feed.ToObject<V4FormatFeedEntry>();
 
-            UpdateCoreToolsReferences(feedEntry.coreTools, coreToolsInfo.ArtifactsDirectory, coreToolsInfo.Version);
+            string linkSuffix = _linkSuffix.ContainsKey(_tag) ? _linkSuffix[_tag] : string.Empty;
```

Comment:
Will this catch v0-prerelease? the suffix list seems to only have the key "v0". Same question for this pattern used elsewhere.

Reply:
The feed generation logic takes what we have in the non-prerelease tags, Updates the core tools and templates references and creates a new pre-release tag. That is why it is referencing `v0` instead of `v0-prerelease`. I matched the feed generated with the latest commit against the previous ones. The output matches the expectation.

Reply:
There is a separate piece of automation that moves the pre-release tags to GA tag

Reply:
Yeah, tracing through, I think that's indeed the case. So this should be fine. We'll find out through some validation run-throughs as we go, regrdless.